src/main/java/ch/njol/skript/lang/ExpressionList.java

```diff
@@ -302,6 +299,23 @@ public Expression<? extends T>[] getExpressions() {
 		return expressions;
 	}
 
+	/**
+	 * Retrieves all expressions, including those nested within any {@code ExpressionList}s.
+	 *
+	 * @return A list of all expressions.
+	 */
+	public List<Expression<? extends T>> getAllExpressions() {
```

Comment:
Where did this end up getting used (if anywhere?)

Reply:
Nowhere at the moment. But, it could be useful, for example, using it in EffCopy (instead of unwrapExpressionList)